### ISSUE TITLE
Can't run keyd-application-manager in verbose mode on Ubuntu

### ISSUE THREAD

**Issue by jonwilts** [rvaiya/keyd#477]
Hi - 

TL;DR - on Ubuntu there seems to be no way to get keyd-application-manager to run in verbose mode.

I'm setting up some application specific mappings on Ubuntu 20.10. In order to do this, I need to know the exact window name that keyd-application-manager looks for. 

Running 'keyd-application-manager -h' tells me that if I were to run 'keyd-application-manager -v' then window names will get logged to ~/.config/keyd/app.log, which is exactly what I want. But this seems impossible on Ubuntu, because stopping the keyd Gnome extension, and then running 'keyd-application-manager -v' from a terminal just restarts the keyd extension, seemingly without triggering verbose mode. 

I know keyd-application-manager is working because I managed to guess '[gnome-terminal]' was the correct window name, and have got some mappings working in Terminal. But there must be a better way than guessing?!

Thank you.
